packages/tocco-test-util/README.md

```diff
@@ -0,0 +1,9 @@
+# Tocco-test-util
+This package contains the most needed mocks for tests.
```

Comment:
i wouldn't reduce this package to mocks. It could also contain some useful utils like helper classes / methods or generic test data. but t